src/Handlebars.Net.Helpers.Core/Enums/Category.cs

```diff
@@ -15,5 +15,6 @@ public enum Category
     Random,
     JsonPath,
     DynamicLinq,
-    Humanizer
+    Humanizer,
+    Boolean
```

Comment:
Can you also update the description in de .csproj to include Boolean ?

Reply:
Done